src/main/java/edu/harvard/iq/dataverse/PermissionServiceBean.java

```diff
@@ -100,6 +100,41 @@ public class PermissionServiceBean {
     @Inject
     DatasetVersionFilesServiceBean datasetVersionFilesServiceBean;
 
+    private static final String LIST_ALL_DATAVERSES_USER_HAS_PERMISSION = "WITH grouplist AS (\n" +
```

Comment:
To avoid all the "\n" and for more maintainability we could use the fancy new text blocks: https://docs.oracle.com/en/java/javase/15/text-blocks/index.html

Reply:
I'll fix that